state-chain/pallets/cf-environment/src/tests.rs

```diff
@@ -16,20 +16,20 @@ fn genesis_config() {
 	});
 }
 
-fn add_utxo_amount(amount: crate::BtcAmount) {
+fn add_utxo_amount(amount: crate::BtcAmount, salt: u32) {
 	Environment::add_bitcoin_utxo_to_list(
 		amount,
 		Default::default(),
-		DepositAddress::new(Default::default(), Default::default()),
+		DepositAddress::new(Default::default(), salt),
 	);
 }
 
 #[test]
 fn test_btc_utxo_selection() {
-	let utxo = |amount| Utxo {
+	let utxo = |amount, salt| Utxo {
 		amount,
 		id: Default::default(),
-		deposit_address: DepositAddress::new(Default::default(), Default::default()),
```

Comment:
why not use a default salt here? utxo selection doesnt care about the salt?

Reply:
because for salt == 0 it is a vault UTXO and the fee is smaller